### PR TITLE
i18n: add/fix ja/cn translation for ew raids

### PR DESCRIPTION
**Pull request by SJoshua**

### CODE REVIEW

**Comment by SJoshua** [comment]
commit message of f48d03f should be `i18n: add ja translation for p3s`.

File: ui/raidboss/data/06-ew/raid/p1s.ts [hunk 1]
```diff
@@ -138,16 +145,22 @@ const triggerSet: TriggerSet<Data> = {
...
           fr: 'Chaînes éloignées sur ${far}',
+          ja: '赤鎖（遠い方）：${far}',
+          cn: '红锁（远）：${far}',
         },
         shacklesOn: {
           en: 'Close: ${close}, Far: ${far}',
           de: 'Nahe: ${close}, Entfernt: ${far}',
           fr: 'Proches : ${close}, Éloignées : ${far}',
+          ja: '紫鎖（近い方）：${close}、赤鎖（遠い方）：${far}',
+          cn: '紫锁（近）：${close}、红锁（远）：${far}',
```
Comment (zhyupe) [inline ui/raidboss/data/06-ew/raid/p1s.ts:163]: It may be confusing to bring red/purple into alarms

File: ui/raidboss/data/06-ew/raid/p1s.ts [hunk 2]
```diff
@@ -309,11 +326,15 @@ const triggerSet: TriggerSet<Data> = {
...
           fr: 'Chaînes à retardement sur ${player}',
+          ja: '時限の魔鎖：${player}',
+          cn: '时限魔锁：${player}',
         },
         shacklesOnYou: {
           en: 'Shackles of Time on YOU',
           de: 'Aspektierende Ketten auf DIR',
           fr: 'Chaînes à retardement sur VOUS',
+          ja: '時限の魔鎖ついた',
+          cn: '时限魔锁：自己',
```
Comment (zhyupe) [inline ui/raidboss/data/06-ew/raid/p1s.ts:337]: suggest: 时间魔锁点名

File: ui/raidboss/data/06-ew/raid/p1s.ts [hunk 3]
```diff
@@ -309,11 +326,15 @@ const triggerSet: TriggerSet<Data> = {
           en: 'Shackles of Time on ${player}',
           de: 'Aspektierende Ketten auf ${player}',
           fr: 'Chaînes à retardement sur ${player}',
+          ja: '時限の魔鎖：${player}',
+          cn: '时限魔锁：${player}',
```
Comment (zhyupe) [inline ui/raidboss/data/06-ew/raid/p1s.ts:330]: suggest: 时间魔锁 点 ${player}

File: ui/raidboss/data/06-ew/raid/p2n.ts [hunk 4]
```diff
@@ -115,24 +115,32 @@ const triggerSet: TriggerSet<Data> = {
           de: 'nördliche Ecken',
           fr: 'Au coin nord',
           ko: '북쪽 모서리',
+          ja: '北の角へ',
+          cn: '去北边的角',
```
Comment (zhyupe) [inline ui/raidboss/data/06-ew/raid/p2n.ts:119]: suggest: 去北边角落

File: ui/raidboss/data/06-ew/raid/p2s.ts [hunk 5]
```diff
@@ -85,6 +85,8 @@ const triggerSet: TriggerSet<Data> = {
         directions: {
           en: 'Back of head',
           de: 'Zur Rückseite des Kopfes',
+          ja: '頭の後ろへ',
+          cn: '去头的后面',
```
Comment (zhyupe) [inline ui/raidboss/data/06-ew/raid/p2s.ts:89]: suggest: 去头前

File: ui/raidboss/data/06-ew/raid/p2s.ts [hunk 6]
```diff
@@ -101,6 +103,8 @@ const triggerSet: TriggerSet<Data> = {
         directions: {
           en: 'Front of head',
           de: 'Zur Vorderseite des Kopfes',
+          ja: '頭の前へ',
+          cn: '去头的前面',
```
Comment (zhyupe) [inline ui/raidboss/data/06-ew/raid/p2s.ts:107]: suggest: 去头后

File: ui/raidboss/data/06-ew/raid/p2s.ts [hunk 7]
```diff
@@ -119,7 +123,7 @@ const triggerSet: TriggerSet<Data> = {
           de: 'Heiler-Gruppen',
           fr: 'Healers en groupes',
           ja: 'ヒラに頭割り',
-          cn: '治疗分摊组',
+          cn: '治疗4:4分摊',
```
Comment (zhyupe) [inline ui/raidboss/data/06-ew/raid/p2s.ts:126]: suggest: 治疗分摊 (prefer) or keep unchanged. 4:4 sounds strange in tts.

**Review by zhyupe — CHANGES_REQUESTED** [review]

**Pull request closed without merging**